Tutorial update (#227)
EMMC-ASBL/tripper: docs/tutorial.md

The API provided by Tripper is modelled after [rdflib], so if you know that library, you will find Tripper rather familiar.

> But there are some differences that you should be aware of.
> Most recognisable:

Reviewer: ```suggestion
But there are some differences that to be aware of.
Most important are:
```

```

### Adding triples to the triplestore

> We can now start to add triples to the triplestore, using the [`add()`] and [`add_triples()`] methods:

Reviewer: ```suggestion
Triples can now be added to the triplestore, using the [`add()`] and [`add_triples()`] methods:
```

```

The function [`en()`] is just a convenient function for adding English literals.
It is equivalent to `tripper.Literal(msg, lang="en")`.

> You can also load triples from a source using the [`parse()`] method.

Reviewer: ```suggestion
Triples can also be added from a source using the [`parse()`] method.
```

load all triples in turtle file `onto.ttl` into the triplestore.

> Similarly you can serialise the triplestore to a string or a file using the [`serialize()`] method:

Reviewer: ```suggestion
Similarly, the triplestore can be serialised to a string or a file using the [`serialize()`] method:
```

```python
ts.serialize("onto2.ttl")  # serialise to file `onto2.ttl`
s = ts.serialize(format="ntriples")  # serialise to string s in ntriples format
```

### Retrieving triples from and querying a triplestore

> A set of convenient functions exists for simple queries, including [`triples()`], [`subjects()`], [`predicates()`], [`objects()`], [`subject_predicates()`], [`subject_objects()`], [`predicate_objects()`] and [`value()`].
> Except for [`value()`], they return the result as generators.

Reviewer: ```suggestion
A set of convenient functions exist for simple queries, including [`triples()`], [`subjects()`], [`predicates()`], [`objects()`], [`subject_predicates()`], [`subject_objects()`], [`predicate_objects()`] and [`value()`].
Except for [`value()`], they return generators.
```

```

The above example works, since the `namespace="https://w3id.org/emmo#"` is resolvable.

> In the case when the IRI in the `namespace` argument is not resolvable, it is possible to supply a resolvable IRI or a reference to a populated [Triplestore] instance via the `triplestore` keyword argument.

Reviewer: ```suggestion
When the IRI in the `namespace` argument is not resolvable, it is possible to supply a resolvable IRI or a reference to a populated [Triplestore] instance via the `triplestore` keyword argument.
```
